compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -437,6 +441,26 @@ private void defineTypeDef(DataInputStream dataInStream) throws IOException {
         }
     }
 
+    private void setInvokableTypeSymbol(BInvokableType invokableType) {
+        BInvokableTypeSymbol tsymbol = (BInvokableTypeSymbol) invokableType.tsymbol;
+        List<BVarSymbol> params = new ArrayList<>();
+        for (BType paramType : invokableType.paramTypes) {
+            BVarSymbol varSymbol = new BVarSymbol(paramType.flags, Names.EMPTY,
+                    this.env.pkgSymbol.pkgID,
+                    paramType, null);
+            params.add(varSymbol);
```

Comment:
if we inline `varSymbol` everything fit in a single line.
`            params.add(new BVarSymbol(paramType.flags, Names.EMPTY, this.env.pkgSymbol.pkgID, paramType, null));
`

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/19976/commits/7432217a84fd35c7ae7682d1e9173e9416f9ccdc